### PR TITLE
adding db lb

### PR DESCRIPTION
**Pull request by EarthmanT**
ansible playbook

lb

update lb gitmodule

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -2,3 +2,15 @@
...
+	url = https://github.com/EarthmanT/mariadb-ansible-galera-cluster.git
+[submodule "db-lb-app/lb/haproxy/roles"]
+	path = db-lb-app/lb/haproxy/haproxy
+	url = https://github.com/EarthmanT/ansible-role-haproxy.git
+[submodule "db-lb-app/app/drupal/droop"]
+	path = db-lb-app/app/drupal/droop
+	url = https://github.com/earthmant/ansible-role-drupal-1.git
+[submodule "db-lb-app/app/drupal/apache"]
+	path = db-lb-app/app/drupal/apache
+	url = https://github.com/earthmant/ansible-role-httpd.git
```
Comment (mabuaisha) [inline .gitmodules]: @EarthmanT I noticed that all the `url`s point to your  own repo, are we going to move them from there or it is ok ?

File: db-lb-app/infrastructure/aws.yaml [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+        GroupName: { concat: [ { get_input: resource_name_prefix }, '-infra-group' ] }
+        Description: db lb app infra group
+        VpcId: { get_capability: [ aws, vpc_id ] }
+
+  subnet:
+    type: cloudify.nodes.aws.ec2.Subnet
+    properties:
+      client_config: *client_config
+      use_external_resource: true
+      resource_id: { get_capability: [ aws, public_subnet_id ] }
```
Comment (mabuaisha) [inline db-lb-app/infrastructure/aws.yaml]: @EarthmanT Why you are using `get_capability` and where `aws` get from here

Comment (EarthmanT) [reply db-lb-app/infrastructure/aws.yaml]: Because this is a feature we want to promote.

File: .gitmodules (hunk 1 above)
Comment (EarthmanT) [reply .gitmodules]: yeah it's ok. We're not going to fork them all to internal repos.

File: db-lb-app/infrastructure/aws.yaml (hunk 2 above)
Comment (mabuaisha) [reply db-lb-app/infrastructure/aws.yaml]: 👍

File: .gitmodules (hunk 1 above)
Comment (mabuaisha) [reply .gitmodules]: ok

**Pull request merged**